### PR TITLE
bdist_rpm script for clusterbuildrpm

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: bdist_rpm.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+if [ ${RELEASE} -gt 1 ]; then
+    SUFFIX=${VERSION}-${RELEASE}
+else
+    SUFFIX=${VERSION}
+fi
+
+git archive --format=tar.gz -o /tmp/slurm-${SUFFIX}.tar.gz --prefix=slurm-${SUFFIX}/ 17.11.ug
+
+mv /tmp/slurm-${SUFFIX}.tar.gz ${HOME}/rpmbuild/SOURCES
```
Comment (stdweird) [inline bdist_rpm.sh]: set this to the `-o` of line above

File: bdist_rpm.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+cp slurm.spec ${HOME}/rpmbuild/SPECS
```
Comment (stdweird) [inline bdist_rpm.sh]: i don't think this is needed. you can run rpmbuild from anywhere

File: bdist_rpm.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+cd ${HOME}/rpmbuild/SPECS
+
+rpmbuild -ba slurm.spec --with=mysql --with=lua --with=hwloc --with=numa
```
Comment (stdweird) [inline bdist_rpm.sh]: i don't like hardocded release versions from spec files:
can you add a
```
gittag=`git log --format=%ct.%h -1`
rpmbuild --define "gittag $gittag"
```
and set in the spec file `Release: %{rel}%{gittag}%{?dist}.ug'

untested, but should work fine

Comment (itkovian) [reply bdist_rpm.sh]: Done.

**Pull request merged**